### DISCUSSION TITLE
Sending Text Message throwing error

### DISCUSSION THREAD

**Discussion by ebadhassanDev** [gabrieldwight/Whatsapp-Business-Cloud-Api-Net#55] [Q&A]
Unsupported post request. Object with ID 'messages' does not exist, cannot be loaded due to missing permissions, or does not support this operation. Please read the Graph API documentation at https://developers.facebook.com/docs/graph-api

I got this error while sending the nromal text message, I've attached all the IDs in program.cs file
How can I do Now I just clone your project.

Thanks

**Comment by gabrieldwight** [comment]
hi, a post request for sending the message does not contain messages, this is what a post request looks like;

```curl
curl -i -X POST `
  https://graph.facebook.com/v17.0/{phone number id}/messages `
  -H 'Authorization: Bearer {your token}' `
  -H 'Content-Type: application/json' `
  -d '{ \"messaging_product\": \"whatsapp\", \"to\": \"{your phone number/test number}\", \"type\": \"template\", \"template\": { \"name\": \"hello_world\", \"language\": { \"code\": \"en_US\" } } }'
```
